contribs/gmf/src/drawing/featureStyleComponent.js

```diff
@@ -162,7 +162,7 @@ exports.Controller_.prototype.handleFeatureSet_ = function(
       keys.push(
         olEvents.listen(
           newFeature,
-          olObject.getChangeEventType(propName),
+          getChangeEventType(propName),
```

Comment:
We should create the string ourselves: `change-${propName}`.
See also the other files below where you should do for ex: 'change-dirty'.